packages/di/src/providers.ts

```diff
@@ -0,0 +1,97 @@
+import { readInjectableMetadata } from './metadata';
+
+export const Type = Function;
+
+/// @ts-ignore
```

Comment:
Why do we need to ignore this?

Reply:
Because `T` the generic type is not used anywhere but we need it later on to do `infer`.